### ISSUE TITLE
Failed tests on macOS

### ISSUE THREAD

**Issue by nepeat** [mvz/email-outlook-message-perl#16]
macOS Version: 11.0 Beta 20A4299v
perl version: v5.28.2
cpan version: `/usr/bin/cpan script version 1.67, CPAN.pm version 2.20`

Installed with `cpan -i Email::Outlook::Message`

Command failure output is as follows.

```
Loading internal logger. Log::Log4perl recommended for better logging
Reading '/Users/nep/.cpan/Metadata'
  Database was generated on Sat, 18 Jul 2020 20:17:03 GMT
Running install for module 'Email::Outlook::Message'
Checksum for /Users/nep/.cpan/sources/authors/id/M/MV/MVZ/Email-Outlook-Message-0.919.tar.gz ok
'YAML' not installed, will not store persistent state
Configuring M/MV/MVZ/Email-Outlook-Message-0.919.tar.gz with Build.PL
Created MYMETA.yml and MYMETA.json
Creating new 'Build' script for 'Email-Outlook-Message' version '0.919'
  MVZ/Email-Outlook-Message-0.919.tar.gz
  /usr/bin/perl Build.PL -- OK
Running Build for M/MV/MVZ/Email-Outlook-Message-0.919.tar.gz
Building Email-Outlook-Message
  MVZ/Email-Outlook-Message-0.919.tar.gz
  ./Build -- OK
Running Build test
t/basics.t ............... ok
t/gpg_signed.t ........... ok
t/internals.t ............ 1/17
#   Failed test at t/internals.t line 66.
#          got: 'text/plain; charset=UTF-8'
#     expected: 'text/plain; charset="UTF-8"'
# Looks like you failed 1 test of 17.
t/internals.t ............ Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/17 subtests
t/plain_jpeg_attached.t .. 1/23
#   Failed test 'Testing content disposition'
#   at t/plain_jpeg_attached.t line 38.
#          got: 'attachment; filename=test.jpg'
#     expected: 'attachment; filename="test.jpg"'
# Looks like you failed 1 test of 23.
t/plain_jpeg_attached.t .. Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/23 subtests
t/plain_uc_unsent.t ...... ok
t/plain_uc_wc_unsent.t ... ok
t/plain_unsent.t ......... ok
t/pod_coverage.t ......... skipped: Test::Pod::Coverage required for testing pod coverage

Test Summary Report
-------------------
t/internals.t          (Wstat: 256 Tests: 17 Failed: 1)
  Failed test:  9
  Non-zero exit status: 1
t/plain_jpeg_attached.t (Wstat: 256 Tests: 23 Failed: 1)
  Failed test:  21
  Non-zero exit status: 1
Files=8, Tests=87,  1 wallclock secs ( 0.03 usr  0.02 sys +  0.76 cusr  0.25 csys =  1.06 CPU)
Result: FAIL
Failed 2/8 test programs. 2/87 subtests failed.
  MVZ/Email-Outlook-Message-0.919.tar.gz
  ./Build test -- NOT OK
//hint// to see the cpan-testers results for installing this module, try:
  reports MVZ/Email-Outlook-Message-0.919.tar.gz
```

**Comment by Jeffrey-Lang** [comment]
Same issue here:

`Created MYMETA.yml and MYMETA.json
Creating new 'Build' script for 'Email-Outlook-Message' version '0.919'
Building Email-Outlook-Message
  MVZ/Email-Outlook-Message-0.919.tar.gz
  ./Build -- OK
'YAML' not installed, will not store persistent state
Running Build test
t/basics.t ............... ok
t/gpg_signed.t ........... ok
t/internals.t ............ 1/17
#   Failed test at t/internals.t line 66.
#          got: 'text/plain; charset=UTF-8'
#     expected: 'text/plain; charset="UTF-8"'
# Looks like you failed 1 test of 17.
t/internals.t ............ Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/17 subtests
t/plain_jpeg_attached.t .. 1/23
#   Failed test 'Testing content disposition'
#   at t/plain_jpeg_attached.t line 38.
#          got: 'attachment; filename=test.jpg'
#     expected: 'attachment; filename="test.jpg"'
# Looks like you failed 1 test of 23.
t/plain_jpeg_attached.t .. Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/23 subtests
t/plain_uc_unsent.t ...... ok
t/plain_uc_wc_unsent.t ... ok
t/plain_unsent.t ......... ok
t/pod_coverage.t ......... skipped: Test::Pod::Coverage required for testing pod coverage

Test Summary Report
-------------------
t/internals.t          (Wstat: 256 Tests: 17 Failed: 1)
  Failed test:  9
  Non-zero exit status: 1
t/plain_jpeg_attached.t (Wstat: 256 Tests: 23 Failed: 1)
  Failed test:  21
  Non-zero exit status: 1
Files=8, Tests=87,  0 wallclock secs ( 0.03 usr  0.02 sys +  0.43 cusr  0.09 csys =  0.57 CPU)
Result: FAIL
Failed 2/8 test programs. 2/87 subtests failed.
  MVZ/Email-Outlook-Message-0.919.tar.gz
  ./Build test -- NOT OK
//hint// to see the cpan-testers results for installing this module, try:
  reports MVZ/Email-Outlook-Message-0.919.tar.gz
Running Build install
  make test had returned bad status, won't install without force`

**Comment by mvz** [comment]
This test failure is due to a recent change in Email::MIME.

**Comment by mvz** [comment]
This is fixed in master.

**Comment by mvz** [comment]
This should be fixed in version 0.920 that has just been released.
